packages/fxa-content-server/app/scripts/views/post_verify/finish_account_setup/set_password.js

```diff
@@ -45,6 +45,7 @@ class SetPassword extends FormView {
   showValidationErrorsEnd() {
     if (this._getPassword() !== this._getVPassword()) {
       const err = AuthErrors.toError('PASSWORDS_DO_NOT_MATCH');
+      this.logError(err);
```

Comment:
we don't need this in sentry, do we?

Reply:
Maybe not?  My guess is that this will have similar rate like our signup flows, but I don't think it would hurt to have it in.

Reply:
we should be getting rid of these kinds of things from sentry, that aren't bugs, not adding more